### ISSUE TITLE
Alpha-defensin positive in joint aspirate

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10689]
**Preferred term label:**
Alpha-defensin positive in joint aspirate

**Synonyms**
cryptdins

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Periprosthetic joint infection (100% specific)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001242
- label: Positive synovial fluid alpha-defensin
- parents: [Positive synonvial fluid infectious agent test (HP:6000581)](https://hpo.jax.org/app/browse/term/HP:6000581)
- pmid: PMID:25631170
- def: Elevation of the the alpha-defensin concentration in synovial aspirate is above the upper limit of normal. The alpha-defensin test is a qualitative immunoassay optimized and validated for synovial fluid. The alpha-defensin protein is an antimicrobial peptide that is naturally released by neutrophils responding to a pathogen in the synovial fluid.
- comment: The alpha-defensin test has been previously demonstrated to be highly accurate in the diagnosis of prosthetic joint infection.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10689
- synonyms:
